docs/src/content/docs/configuration/localizations.md

```
@@ -0,0 +1,93 @@
+---
+title: Localizations
+description: Language and region settings
+---
+
+Flutter Server Box supports 12+ languages with full localization.
+
+## Supported Languages
+
+| Language | Status |
+|----------|--------|
```

markdownlint-cli2 (0.20.0)

Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)